src/integrations/watchers/addon-update-watcher.php

```diff
@@ -3,36 +3,60 @@
 namespace Yoast\WP\SEO\Integrations\Watchers;
 
 use Yoast\WP\SEO\Conditionals\Admin_Conditional;
+use Yoast\WP\SEO\Helpers\Product_Helper;
 use Yoast\WP\SEO\Integrations\Integration_Interface;
 
 /**
- * Enables Yoast add-on auto updates when Yoast SEO is enabled and the other way around.
+ * Enables Yoast add-on auto updates when Yoast SEO (Premium) is enabled and the other way around.
  *
  * Also removes the auto-update toggles from the Yoast SEO add-ons.
  */
 class Addon_Update_Watcher implements Integration_Interface {
 
 	/**
-	 * ID string used by WordPress to identify the free plugin.
+	 * ID string used by WordPress to identify the Yoast SEO plugin.
 	 *
 	 * @var string
 	 */
 	const WPSEO_FREE_PLUGIN_ID = 'wordpress-seo/wp-seo.php';
 
+	/**
+	 * ID string used by WordPress to identify the Yoast SEO Premium plugin.
+	 *
+	 * @var string
+	 */
+	const WPSEO_PREMIUM_PLUGIN_ID = 'wordpress-seo-premium/wp-seo-premium.php';
+
 	/**
 	 * A list of Yoast add-on identifiers.
 	 *
 	 * @var string[]
 	 */
 	const ADD_ONS = [
-		'wordpress-seo-premium/wp-seo-premium.php',
 		'wpseo-video/video-seo.php',
-		'wordpress-seo-local/local-seo.php',
+		'wpseo-local/local-seo.php',
```

Comment:
Has this been changed in the meanwhile?

Reply:
If it has been change the same change should be applied in class `WPSEO_Plugin_Availability`

Reply:
> If it has been change the same change should be applied in class WPSEO_Plugin_Availability

Can you create a quick PR or issue to do this? These changes will ultimately get reverted for 16.0, so changing it here is not advisable 😛 .